frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap

```diff
@@ -107,7 +107,7 @@ Object {
                     >
                       <label
                         class="usa-label"
-                        for="120"
```

Comment:
Huh, maybe I'm missing something obvious but how come so many of the HTML element ids changed here?

Reply:
Yeah I have no idea

Reply:
I think this might actually be why I have consistently had issues with snapshots not matching.  I've always run them via the intellij interface.  Reran them just from the CLI and now the ids are back to the way they were.